<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(1, 72, 68, 255), stop:1 rgba(0, 225, 210, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget {
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 62, 128, 255), stop:1 rgba(98, 225, 255, 255));
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>800</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#stackedWidget {
background-image: url(:/background.jpeg);
}</string>
    </property>
    <widget class="QWidget" name="StartPage">
     <property name="styleSheet">
      <string notr="true">#StartPage{
background-image: url(:/background.jpeg);
}</string>
     </property>
     <widget class="QLabel" name="main_label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>711</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:reflect, x1:0.488, y1:0.006, x2:1, y2:0, stop:0 rgba(11, 121, 214, 200), stop:0.648241 rgba(11, 121, 214, 120), stop:1 rgba(0, 0, 0, 0));
font: 100pt;
</string>
      </property>
      <property name="text">
       <string>Qute Chat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>400</y>
        <width>481</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox {
	background-color: rgba(11, 121, 214, 120);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>411</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="StartLogButton">
          <property name="styleSheet">
           <string notr="true">#StartLogButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 116, 213);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
	font: 20pt;
	min-height: 2em;
}

#StartLogButton:hover {

	background-color: rgb(0, 131, 214);
	border-color: rgb(210, 243, 255);
}</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StartRegButton">
          <property name="styleSheet">
           <string notr="true">#StartRegButton {
	color: rgb(255, 255, 255);	
	background-color: rgb(0, 116, 213);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
	font: 20pt;
	min-height: 2em;
}

#StartRegButton:hover { 
	
	background-color: rgb(0, 131, 214);
	border-color: rgb(210, 243, 255);
}</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="LogPage">
     <property name="styleSheet">
      <string notr="true">#LoginPage{
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 62, 128, 255), stop:1 rgba(98, 225, 255, 255));
}</string>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>1061</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_3 {
	background-color: rgba(11, 121, 214, 120);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>1001</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item row="1" column="1">
         <widget class="QLineEdit" name="log_password_edit">
          <property name="styleSheet">
           <string notr="true">font: 25pt;
background-color: rgb(210, 243, 255);
border-radius: 10px;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="log_password_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
	font: 25pt;</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="log_nickname_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
	font: 25pt;</string>
          </property>
          <property name="text">
           <string>Nickname:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="log_nickname_edit">
          <property name="styleSheet">
           <string notr="true">font: 25pt;
background-color: rgb(210, 243, 255);
border-radius: 10px;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>580</y>
        <width>371</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="LogLogButton">
         <property name="styleSheet">
          <string notr="true">#LogLogButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 116, 213);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
	font: 20pt;
	min-height: 2em;
}

#LogLogButton:hover {

	background-color: rgb(0, 131, 214);
	border-color: rgb(210, 243, 255);
}</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LogReturnButton">
         <property name="styleSheet">
          <string notr="true">#LogReturnButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 116, 213);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
	font: 20pt;
	min-height: 2em;
}

#LogReturnButton:hover {

	background-color: rgb(0, 131, 214);
	border-color: rgb(210, 243, 255);
}</string>
         </property>
         <property name="text">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="RegPage">
     <property name="styleSheet">
      <string notr="true">#RegisterPage{
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 62, 128, 255), stop:1 rgba(98, 225, 255, 255));
}</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>1151</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_2 {
	background-color: rgba(11, 121, 214, 120);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1109</width>
         <height>247</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="reg_nickname_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
	font: 25pt;</string>
          </property>
          <property name="text">
           <string>Nickname:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="reg_confirm_password_edit">
          <property name="styleSheet">
           <string notr="true">font: 25pt;
background-color: rgb(210, 243, 255);
border-radius: 10px;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="reg_password_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
	font: 25pt;</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="reg_confirm_password_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
	font: 25pt;</string>
          </property>
          <property name="text">
           <string>Confirm password:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="reg_nickname_edit">
          <property name="styleSheet">
           <string notr="true">font: 25pt;
background-color: rgb(210, 243, 255);
border-radius: 10px;
</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="reg_password_edit">
          <property name="styleSheet">
           <string notr="true">font: 25pt;
background-color: rgb(210, 243, 255);
border-radius: 10px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>490</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="RegRegButton">
         <property name="styleSheet">
          <string notr="true">#RegRegButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 116, 213);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
	font: 20pt;
	min-height: 2em;
}

#RegRegButton:hover {

	background-color: rgb(0, 131, 214);
	border-color: rgb(210, 243, 255);
}</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RegReturnButton">
         <property name="styleSheet">
          <string notr="true">#RegReturnButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 116, 213);
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: rgb(73, 168, 190);
	font: 20pt;
	min-height: 2em;
}

#RegReturnButton:hover {

	background-color: rgb(0, 131, 214);
	border-color: rgb(210, 243, 255);
}</string>
         </property>
         <property name="text">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ChatPage">
     <property name="styleSheet">
      <string notr="true">#ChatPage{

}</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>1121</width>
        <height>751</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>865</number>
         </property>
         <item>
          <widget class="QLabel" name="chat_users_label">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 25pt;
background-color: qlineargradient(spread:reflect, x1:0.488, y1:0.006, x2:1, y2:0, stop:0 rgba(106, 186, 255, 180), stop:0.648241 rgba(114, 149, 219, 180), stop:1 rgba(0, 0, 0, 0));</string>
           </property>
           <property name="text">
            <string>Users</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>9</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableView" name="chat_tableview">
             <property name="styleSheet">
              <string notr="true">QTableView {

font: 15pt;
background-color: rgb(210, 243, 255);
border-radius: 10px;
border-style: outset;
border-width: 3px;
border-color: rgb(0, 94, 131);

}

 QScrollBar:vertical {

     background: rgba(143, 210, 236, 70);

 }

 QScrollBar::handle:vertical {

     background: rgb(128, 197, 224);
     min-height: 20px;
	  border-radius: 5px;

 }


QScrollBar::add-line:horizontal {
      border: none;
      background: none;
}

QScrollBar::sub-line:horizontal {
      border: none;
      background: none;
}</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QListView" name="chat_userlist_view">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 20pt;
background-color: rgb(210, 243, 255);
border-radius: 10px;
border-style: outset;
border-width: 3px;
border-color: rgb(0, 94, 131);
</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="chat_message_edit">
             <property name="styleSheet">
              <string notr="true">font: 15pt;
background-color: rgb(210, 243, 255);
border-radius: 10px;
border-style: outset;
border-width: 3px;
border-color: rgb(0, 94, 131);

	min-height: 1.2em;
</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="ChatSendButton">
             <property name="styleSheet">
              <string notr="true">#ChatSendButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 116, 213);
    border-style: outset;
    border-width: 3px;
    border-radius: 10px;
	border-color: rgb(0, 56, 104);
	font: 15pt;
	min-height: 1.4em;
}

#ChatSendButton:hover {

	background-color: rgb(0, 131, 214);
	border-color: rgb(210, 243, 255);
}</string>
             </property>
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
